torch/csrc/jit/codegen/cuda/expr_simplifier.cpp

```diff
@@ -711,19 +944,82 @@ namespace prove {
 // - x can be either zero or non-zero, it is just a symbolic number that depends
 // - x is zero
 
-bool isNonZero(Val* value) {
+bool isNonNegative(Val* value) {
```

Comment:
No need to read this, it is just a temporary solution allowing me to create non-zero variables in order to trigger `simplifyDivisibleDivMod` in unit tests. This will be replaced by https://github.com/csarofeen/pytorch/pull/2273